src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -93,9 +93,9 @@ const SidebarSidecarLayoutContent = ({
         >
           <SidebarContent />
         </motion.div>
-        <div className={s.mainArea}>
-          <div className={s.main}>
-            <main id="main">
+        <div className={s.mainAreaAndFooter}>
```

Comment:
This `.mainAreaAndFooter` element creates a flex "column" layout, with two rows:
- first row: main area + sidecar, _cannot_ have `overflow: hidden` because this messes with `sidecar`'s `position: sticky`
- second row: footer + footerSpaceForSidecar, _must_ have `overflow: hidden` to prevent footer top border from overflowing